syntax_sugar/infix.py

```diff
@@ -45,19 +47,19 @@ def __iter__(self):
     
     def __next__(self):
         if self.type == 'number':
-            if self.curr <= self.end:
+            if self.curr == self.end + self.step:
```

Comment:
I also thought about that, what do you think of a syntax like:
```
list(1 /to(3)/ 10) == [1, 4, 7, 10]
list(10 /to(3)/ 1) == [10, 7, 4, 1]
```

Thus, the parameter passed to `to` would define the absolute value of the interval